build-tools/automation/azure-pipelines.yaml

```diff
@@ -99,209 +99,6 @@ stages:
   displayName: Smoke Tests
   dependsOn: mac_build
   jobs:
-  # Check - "Xamarin.Android (macOS > Tests > APKs Classic)"
-  - job: mac_apk_tests_legacy
```

Comment:
I'm not sure that we want to remove this job entirely just yet, I think it would be preferable to port more of these test suites and get them running on the .NET test jobs first?

Reply:
Yeah, I was wondering about that, that's why I made it a separate commit to make it easier to revert.

Reply:
OTOH, the tests themselves (or YAML templates) aren't removed, so perhaps it's fine to remove the YAML invocation of the tests and restore them once they will have been ported?